### PR TITLE
changes for SAAS-900, SAAS-899 and SAAS-895

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

**Review by Nailya — APPROVED** [review]

**Review by Nailya — DISMISSED** [review]

File: docs/connect-pmm.md [hunk 1]
```diff
@@ -39,23 +39,29 @@ To connect your PMM server to Percona Platform, generate a personal access token
...
+To confirm that you have successfully connected the server and check the list of all servers currently connected to an organization, go to [Percona Platform](https://portal.percona.com) **> PMM Instances**. 
 
 ### Disconnect a PMM instance
 
 Disconnect a PMM instance when you want to unlink it from your Percona Platform organization or stop monitoring it there.
 
 To disconnect a PMM server, go to > <i class="uil uil-cog"></i> **Configuration > Settings > Percona Platform** and click **Disconnect**.
 
-To confirm that the server disconnected successfully, go to Percona Platform > **Dashboard** tab > **View instances**. This displays the **PMM instances** page where you can check the list of servers currently connected to your Platform organization.
+To confirm that the server disconnected successfully, go to Percona Platform > **View instances**. This displays the **PMM instances** page where you can check the list of servers currently connected to your Platform organization.
```
Comment (Hlmn-Rmr) [inline docs/connect-pmm.md:58]: Is it Percona Platform > View instances or Percona Platform > PMM instances? If this refers to the option on the Portal left hand side menu, then it is PMM instances, please double-check

File: docs/connect-pmm.md [hunk 2]
```diff
@@ -39,23 +39,29 @@ To connect your PMM server to Percona Platform, generate a personal access token
...
+
+### Remove entries from the list of connected PMM instances
+
+If you want to remove a PMM server from the list of connected PMM instances without disconnecting it from Percona Platform, go to go to [Percona Platform](https://portal.percona.com) ** > View Instances** and click the Remove icon ![Remove icon](images/remove-instance.png) next to the server you want to remove from the list.
```
Comment (Hlmn-Rmr) [inline docs/connect-pmm.md:62]:
```suggestion
If you want to remove a PMM server from the list of connected PMM instances without disconnecting it from Percona Platform, go to [Percona Platform](https://portal.percona.com) ** > View Instances** and click the Remove icon ![Remove icon](images/remove-instance.png) next to the server you want to remove from the list.
```

Comment (Hlmn-Rmr) [reply docs/connect-pmm.md:62]: Also, similar to my previous comment... is it Percona Platform > View instances or Percona Platform > PMM Instances ?

**Review by Nailya — APPROVED** [review]

File: docs/registration-login.md [hunk 3]
```diff
@@ -13,9 +13,10 @@ Percona Platform can also use your organization credentials to log you in. To en
 If you don't have a Percona Account yet, you can create one straight from the **Sign in** form:
 
 1. Go to the [Percona Platform Log in page](https://portal.percona.com/).
-2. Scroll down to the **Don't have an account? Create one** link at the bottom of the form.
-3. Fill in the fields on the **Crate New Account** form, and click  **Create** once you’ve accepted the Terms of Service and Privacy Policy. 
+2. Scroll down to the **Don't have an account? Create one** link at the bottom of the form. Alternatively, you can register with a social identity provider if you have a Google or GitHub account that is not yet associated with a Percona Platform accont. To do this, click **Sign in with Google** or **Sign in with GitHub**.
```
Comment (Hlmn-Rmr) [inline docs/registration-login.md:16]:
```suggestion
2. Scroll down to the **Don't have an account? Create one** link at the bottom of the form. Alternatively, you can register with a social identity provider if you have a Google or GitHub account that is not yet associated with a Percona Platform account. To do this, click **Sign in with Google** or **Sign in with GitHub**.
```

**Pull request merged**